rest/data/subscription_test.go

```diff
@@ -72,3 +73,76 @@ func TestGetSubscriptions(t *testing.T) {
 	assert.EqualError(err, "400 (Bad Request): no subscription owner provided")
 	assert.Len(apiSubs, 0)
 }
+
+func TestCopyProjectSubscriptions(t *testing.T) {
+	assert := assert.New(t)
+	require := require.New(t)
```

Comment:
I would avoid this. I know we do it a lot in other places, but this will cause a reporting error in the test, as the *testing.T that is passed to the subtest isn't the same one that we're asserting with which would be bad.